products/emberjs.md

```diff
@@ -16,27 +16,44 @@ auto:
 -   npm: ember-source
 
 releases:
--   releaseCycle: "4"
+-   releaseCycle: "4.12"
+    releaseDate: 2023-04-03
     support: true
     eol: false
     latest: "4.12.0"
     latestReleaseDate: 2023-04-03
-    releaseDate: 2021-11-15
+
+-   releaseCycle: "4.8"
+    lts: true
+    releaseDate: 2022-11-28
+    support: 2023-08-07
+    eol: 2023-12-11
+    latest: "4.8.4"
+    latestReleaseDate: 2022-12-04
```

Comment:
Will be automatically updated to 2022-12-13, see https://github.com/endoflife-date/release-data/blob/74489af2880e199b375cdb570e5c0663d2996421/releases/emberjs.json#L178.

Reply:
Fixed with 25b7ec5